inf-elixir.el

```diff
@@ -81,6 +81,12 @@ printed instead."
   :type 'buffer
   :group 'inf-elixir)
 
+(defcustom inf-elixir-on-send-switch-to-repl t
+  "If `t' switch to the corresponding repl window on any send command."
+
+  :type 'buffer
```

Comment:
The `:type` field is used by Emacs' Customization system to show appropriate values for the variable, and I think the "type" for this field should be `boolean` (like `inf-elixir-prefer-umbrella`).

```suggestion
  :type 'boolean
```